tasks/postgresql.yml

```diff
@@ -3,7 +3,7 @@
   apt:
     pkg:
       - libpq-dev
-      - python-psycopg2
+      - python3-psycopg2
```

Comment:
Can we make this configurable? I'm assuming servers with no python3 will run into this issue